lib/autoproj/package_set.rb

```diff
@@ -168,7 +168,7 @@ def initialize(
             @version_control = Array.new
             @overrides = Array.new
             @raw_local_dir = raw_local_dir
-            @default_importer = VCSDefinition.from_raw(type: 'none')
+            @default_importer = VCSDefinition.from_raw({ type: 'none' })
```

Comment:
Is this really necessary? I think rubocop even complains about this

Reply:
It's actually one of the major changes. `from_raw`'s signature is `from_raw(specs, <keyword arguments>)`. 2.7 pushes (through a warning) to disambiguate the specs (which is a hash) and the keyword arguments by explicitly adding hash marks.